sympy/functions/elementary/trigonometric.py

```diff
@@ -1200,6 +1208,10 @@ def _eval_rewrite_as_sqrt(self, arg, **kwargs):
             return None
         return y
 
+    def _eval_rewrite_as_besselj(self, arg, **kwargs):
+        from sympy.functions.special.bessel import besselj
+        return (besselj(S.Half, arg))/(besselj(-S.Half, arg))
```

Comment:
The brackets here are not needed.

Reply:
There are other places with redundant brackets as well (I should not need to point out each case individually).